components/main.css

```diff
@@ -0,0 +1,7 @@
+@import url('./image.css');
+@import url('./avatar.css');
+@import url('./alert.css');
+@import url('./badge.css');
+@import url('./button.css');
+@import url('./utility.css');
+@import url('./root.css');
```

Comment:
root.css should be on top , but do check , need to clear this!!

Reply:
checked online declaring root.css at top is the best practice!!